src/Uno.UI/UI/Xaml/Controls/Border/BorderLayerRenderer.wasm.cs

```diff
@@ -119,6 +98,19 @@ public static void SetBorder(UIElement element, Thickness thickness, Brush brush
 						break;
 				}
 			}
+
+			if (cornerRadius == CornerRadius.None)
+			{
+				element.ResetStyle("border-radius", "overflow");
+			}
+			else
+			{
+				var outer = cornerRadius.GetRadii(element.RenderSize, thickness).Outer;
+				var borderRadius = $"{outer.TopLeft.X}px {outer.TopRight.X}px {outer.BottomRight.X}px {outer.BottomLeft.X}px / {outer.TopLeft.Y}px {outer.TopRight.Y}px {outer.BottomRight.Y}px {outer.BottomLeft.Y}px";
```

Comment:
Does this get recalculated when render size changes (render scale transform for example)? I remember I found the border layer renderer on WASM wasn't refreshed in such case, which meant the corner radius could get incorrect, but maybe it has changed since

Reply:
I think it might not work for this specific case. Do we want that in this PR or in a follow up?

Reply:
I guess it is mostly an edge case, but needs to be addressed - but I am fine with separate PR

Reply:
There is a test that is failing because of this actually. That is because the CornerRadius update happens before arrange, so RenderSize is zero at that time. So after arrange, the CornerRadius doesn't show up at all.

I'll address in this PR.